packages/zarith/zarith.1.7a/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+name: "zarith"
+version: "1.7"
```

Comment:
```suggestion
```
This line makes the package not work. Also could you use the `-1` suffix convention instead of `1.7a`